src/components/magazine/MagazineList/TabMagazine.tsx

```diff
@@ -56,14 +75,26 @@ const TabMagazine = ({ magazines, selectItem, onScrollDown }: TabMagazineProps)
                 `
               }
             >
-              <Link href={`/magazine/${magazine.magazine_id}`}>
-                <Photo
-                  blur={<PhotoSelect enabled={selectItem} />}
-                  src={magazine.cover_url}
-                  width={'196px'}
-                  height={'255px'}
-                />
-              </Link>
+              {boolOption ? (
+                <div>
+                  <Photo
+                    blur={<PhotoSelect enabled={selectItem} />}
+                    src={magazine.cover_url}
+                    width={'196px'}
+                    height={'255px'}
+                    onClick={() => selectMagazineItems(magazine.magazine_id)}
+                  />
+                </div>
+              ) : (
+                <Link href={`/magazine/${magazine.magazine_id}`}>
+                  <Photo
+                    blur={<PhotoSelect enabled={selectItem} />}
+                    src={magazine.cover_url}
+                    width={'196px'}
+                    height={'255px'}
+                  />
+                </Link>
+              )}
```

Comment:
똑같은 `<Photo />` 컴포넌트가 반복되고 있네요!
```suggestion
            const Root = option ? <div /> : <Link href={`/magazine/${magazine.magazine_id}`} />;

          {cloneElement(Root, {
                  children: (
                    <Photo
                      blur={<PhotoSelect enabled={selectItem} />}
                      src={magazine.cover_url}
                      width={'196px'}
                      height={'255px'}
                      onClick={() => selectMagazineItems(magazine.magazine_id)}
                    />
                  ),
                })}
```
이런식으로도 해결 가능해 보입니다!

Reply:
cloneElement 사용해서 저렇게 줄일 수 있는건 오늘 처음 알았네요!!
앞으로도 잘 참고할게요! 
map 안에서 돌아가고 있는 magazine_id 때문에, Root 선언이 안 될 것 같아서 일단 원래코드 사용하겠습니당!😭